package.json

```diff
@@ -12,7 +12,14 @@
     "react": "16.8.3",
     "react-dom": "16.8.3",
     "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
-    "react-native-web": "^0.11.7"
+    "react-native-gesture-handler": "^1.5.2",
```

Comment:
Expo throws a warning that some dependencies (react-native-gesture-handler and react-native-reanimated) have incompatible versions. Apparently the ones that we use are too high and Expo can't handle them yet. We should install such stuff with `expo install` instead of `npm install` or `yarn add`